api-specs/Gateway-EE/3.3/kong-ee-3.3.json

```diff
@@ -1449,7 +1482,10 @@
                     }
                   }
                 ],
-                "type": "array"
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
```

Comment:
The definition of `portal_developer_meta_fields` does not look quite correct according to the definition on the [Enterprise Gateway side](https://github.com/Kong/kong-ee/blob/bf1c39d6e2f194e94b8deead828586a99275dd50/kong/db/schema/entities/workspaces.lua#L70-L71).

`portal_developer_meta_fields` here should be a plain string that contains a serialized JSON array of objects, instead of an array of strings.